### ISSUE TITLE
sbt-dotty looks for wrong doctool jar since bump to 3.0.1-RC1

### ISSUE THREAD

**Issue by griggt** [lampepfl/dotty#11940]
It's looking for `scala3-doc` rather than `scaladoc`:

```
sbt:scala3-simple> doc
[error] Couldn't retrieve `org.scala-lang:scala3-doc:3.0.1-RC1-bin-SNAPSHOT` : Error downloading org.scala-lang:scala3-doc_3.0.1-RC1:3.0.1-RC1-bin-SNAPSHOT
[error]   Not found
[error]   not found: /home/tgrigg/.ivy2/local/org.scala-lang/scala3-doc_3.0.1-RC1/3.0.1-RC1-bin-SNAPSHOT/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scala-lang/scala3-doc_3.0.1-RC1/3.0.1-RC1-bin-SNAPSHOT/scala3-doc_3.0.1-RC1-3.0.1-RC1-bin-SNAPSHOT.pom.
[error] (doc / scalaInstance) Couldn't retrieve `org.scala-lang:scala3-doc:3.0.1-RC1-bin-SNAPSHOT` : Error downloading org.scala-lang:scala3-doc_3.0.1-RC1:3.0.1-RC1-bin-SNAPSHOT
```

The hardcoded `3.0.0` looks dubious:

https://github.com/lampepfl/dotty/blob/5bb0e92b33cd96ac96d394068165b38e4ce5b035/sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala#L375-L378

**Comment by abgruszecki** [comment]
@smarter simply checking if `v` starts with `3` should work, no?

**Comment by smarter** [comment]
Yes I think so.

**Comment by abgruszecki** [comment]
Fixed in #11946.
